common/src/main/java/com/wynntils/features/chat/ChatTabsFeature.java

```diff
@@ -50,6 +50,9 @@ public class ChatTabsFeature extends Feature {
             new ChatTab("Private", false, "/r  ", Sets.newHashSet(RecipientType.PRIVATE), null),
             new ChatTab("Shout", false, null, Sets.newHashSet(RecipientType.SHOUT), null))));
 
+    @Persisted(i18nKey = "feature.wynntils.chatTabs.oldTabHotkey")
```

Comment:
```suggestion
    @Persisted
```

No need to specify the key, unless you want something different from the default key.